plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -543,6 +542,15 @@ def create_topoconfig_file(links_info_dict, ndt_file_path, patterns,
             if peer_device and peer_port:
                 link_key_peer = "%s___%s" % (peer_device, peer_port)
                 peer_port_guid = links_info_dict.get(link_key_peer)
+                if not peer_port_guid:
+                    # unwired link: incorrect peer port number. Such link should
+                    # not be written into topoconfig file - if will cause opensm malfunction
+                    # such line should not appear in topoconfig - continue
+                    error_message = "Failed to detect peer port GUID. Skip link in topoconfig file: peer device: {} peer port: {}".format(peer_device, peer_port)
+                    logging.error(error_message)
+                    file_creation_failed = True
+                    failed_lable_conversion.append(port_key.replace("___", " Port:"))
+                    continue
```

Comment:
are you sure we want to continue in case of such link? doesn't it make sense to exit and report an error? because I suppose the whole NDT file should be correct, and if there is an error - user should fix it before proceeding..

Reply:
according to architecture we should allow used to deploy topoconfig even if there are problems during verification.
so to avoid opensm failure decided not to include links from NDT with problems in topoconfig file. UI will send notification that it is on user responsibility to deploy topoconfig when verification failed